tests/fixtures/migrations/relationships.php

```diff
@@ -13,12 +13,16 @@ class CreateCommentsTable extends Migration
      */
     public function up()
     {
+        Schema::disableForeignKeyConstraints();
```

Comment:
I'm curious why this caused a change in the "after" fixture. I would have expected it to be toggled already given the previous `app->config->set` in the existing test...

Reply:
Because the `draft.yml` file for that test fixture was updated to include `config:`, all test that use that same draft file, now update their configuration before generating code.

Prior to this patch, the configuration was only applied explicitly during test setup and generated the same code: 
`/tests/fixtures/migrations/relationships-constraints.php`